src/Video/ShadowPass.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "linking.hpp"
+
+namespace Video {
+    class ShadowPass {
+        public:
+        VIDEO_API ShadowPass();
+        VIDEO_API ~ShadowPass();
```

Comment:
Missing documentation.

Reply:
Fixed

Reply:
We still lack documentation for the destructor and the class itself.

Reply:
Fixed